master/reference/node/configuration.md

```diff
@@ -27,6 +27,8 @@ The `calico/node` container is primarily configured through environment variable
 | CALICO_IPV4POOL_CIDR | The IPv4 Pool to create if none exists at start up. It is invalid to define this variable and NO_DEFAULT_POOLS. | IPv4 CIDR | 192.168.0.0/16 |
 | CALICO_IPV6POOL_CIDR | The IPv6 Pool to create if none exists at start up. It is invalid to define this variable and NO_DEFAULT_POOLS. | IPv6 CIDR | fd80:24e2:f998:72d6::/64 |
 | CALICO_IPV4POOL_IPIP | IPIP Mode to use for the IPv4 POOL created at start up. | off, always, cross-subnet | off |
+| CALICO_IPV4POOL_NAT_OUTGOING | Controls NAT Outgoing `nat-outgoing` for the IPv4 IPPOOL created at start up. | boolean | true |
```

Comment:
We haven't referenced the specific flag in the Resource in other descriptions so I think you should remove the `nat-outgoing` from both of these entries.
nit: `IPv4 IPPOOL` should be `IPv4 Pool`